tensorboard/webapp/widgets/data_table/data_table_component.ng.html

```diff
@@ -32,21 +50,21 @@
           </td>
           <ng-container *ngFor="let header of headers;">
             <td [ngSwitch]="header">
-              <span *ngSwitchCase="ColumnHeaders.VALUE_CHANGE">
+              <div *ngSwitchCase="ColumnHeaders.VALUE_CHANGE" class="cell">
                 <ng-container
                   *ngTemplateOutlet="arrow; context: {$implicit:runData.VALUE_CHANGE}"
                 ></ng-container>
                 {{ getFormattedDataForColumn(header, runData) }}
-              </span>
-              <span *ngSwitchCase="ColumnHeaders.PERCENTAGE_CHANGE">
+              </div>
+              <div *ngSwitchCase="ColumnHeaders.PERCENTAGE_CHANGE" class="cell">
                 <ng-container
                   *ngTemplateOutlet="arrow; context: {$implicit:runData.PERCENTAGE_CHANGE}"
                 ></ng-container>
                 {{ getFormattedDataForColumn(header, runData) }}
-              </span>
-              <span *ngSwitchDefault
-                >{{ getFormattedDataForColumn(header, runData) }}</span
-              >
+              </div>
+              <div *ngSwitchDefault class="cell extra-right-padding">
```

Comment:
I also want to reconstruct the html to remove the repeated cell class. Yet this is not the requirement for this PR.